src/ucs/datastruct/ptr_array.c

```diff
@@ -20,23 +20,18 @@
 #define UCS_PTR_ARRAY_INITIAL_SIZE  8
 
 
-static inline int ucs_ptr_array_is_free(ucs_ptr_array_t *ptr_array, unsigned idx)
+static inline int
+ucs_ptr_array_is_free(ucs_ptr_array_t *ptr_array, unsigned element_index)
 {
-    return (idx < ptr_array->size) &&
-            __ucs_ptr_array_is_free(ptr_array->start[idx]);
+    return (element_index < ptr_array->size) &&
+            __ucs_ptr_array_is_free(ptr_array->start[element_index]);
 }
 
-static inline uint32_t ucs_ptr_array_placeholder_get(ucs_ptr_array_elem_t elem)
+static inline uint32_t
+ucs_ptr_array_size_free_get(ucs_ptr_array_elem_t elem)
```

Comment:
what is this function doing? need better name (or some doc)
also a general not: pls use UCS_F_ALWAYS_INLINE for fast path
`ucs_ptr_array_size_free_get_free_ahead`

Reply:
Fixed.
Renamed the function to ucs_ptr_array_size_free_get_free_ahead().